packages/api/src/compile-schemas.js

```diff
@@ -72,7 +72,17 @@ const data = _.merge({}, apiData, dbData);
   const indexPath = path.resolve(validatorDir, "index.js");
   write(indexPath, indexStr);
 
-  const typeStr = types.join("\n");
+  const typeDefinition = `export type InputCreatorId =
```

Comment:
So weird that this was necessary. Did you get to understand why it was being generated twice?